config/express.js

```diff
@@ -70,7 +70,7 @@ module.exports.http = {
       clientID: Reddit.data.clientID,
       clientSecret: Reddit.data.clientIDSecret,
       callbackURL: Reddit.data.redirectURL,
-      scope: "flair,modflair"
+      scope: "flair,modflair,modcontributors,wikiread,wikiedit,read,modposts"
```

Comment:
It's not necessary actually. Alrighty, add a link to the moderator dropdown to add in the extra scope, remove it all from here, and check it works.